services/gem/gem-owner.service.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const Joi = require('joi')
+
+const { BaseJsonService } = require('../base')
+const { floorCount: floorCountColor } = require('../../lib/color-formatters')
+
+module.exports = class GemOwner extends BaseJsonService {
+  async handle({ user }) {
+    const url = `https://rubygems.org/api/v1/owners/${user}/gems.json`
+    const json = await this._requestJson({
+      url,
+      schema: Joi.array(),
```

Comment:
Looks like this one is pending ^^

Reply:
I intentionally left this because in this badge we are not using any of the data in the array - we are just counting the number of elements in it. As long the response is an array we're fine. I've moved the declaration out of the class and made it `required()` in 410b329 though. I don't think we need to be any more specific